components/atoms/LanguagePill/LanguagePill.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import Image from "next/legacy/image";
```

Comment:
Use non-legacy image

```suggestion
import Image from "next/image";
```